packages/fiori/src/themes/base/SideNavigation-parameters.css

```diff
@@ -71,8 +71,6 @@
 	--_ui5_side_navigation_item_collapsed_padding_right: 0;
 	--_ui5_side_navigation_action_item_collapsed_padding: 0;
 	--_ui5_side_navigation_item_collapsed_unselectable_padding: 0;
-	--_ui5_side_navigation_action_item_border_hover: var(--sapButton_BorderWidth) solid var(--sapButton_Hover_BorderColor);
```

Comment:
Can't we just leave the themable parametrs, and remove horizon_hcb and and horizon_hcw